src/Makefile.am

```diff
@@ -180,7 +180,6 @@ soufflepublic_HEADERS = \
         Brie.h                                    \
         BTree.h                                   \
         CompiledIndexUtils.h                      \
-        CompiledRecord.h                          \
```

Comment:
RecordTable.h is needed in the public headers, as the replacement for CompiledRecord.h

Reply:
Also please check the file ```configure.ac``` whether RecordTable.h is listed there.